css/css-fonts/WEB_FEATURES.yml

```diff
@@ -0,0 +1,13 @@
+features:
+- name: font-palette
+  files:
+  - font-palette.html
+  - font-palette-*
+  - palette-values-rule-*
+- name: font-synthesis
+  files:
+  - font-synthesis-*
+- name: font-variant-alternates
+  files:
+  - alternates-order.html
+  - font-variant-alternates-*
```

Comment:
Is there any particular reason for which "features" are covered here and which aren't? It seems like there are a bunch more of these tests that could be mapped to a CSS feature; e.g. there are tests in this directory for `font-size-adjust`, `font-weight`, `font-stretch`, other subproperties of `font-variant-*`, and more. So offhand this looks quite incomplete?

Reply:
These are the font-related ones that are in https://github.com/web-platform-dx/web-features so far, and it is incomplete. `font-size-adjust` is in review at https://github.com/web-platform-dx/web-features/pull/748 and I need to come back to it. In https://github.com/web-platform-dx/web-features/issues/666 we're tracking making web-features more complete with various burndown lists. Feel free to comment there, file issues, or send PRs if there are specific features you'd like to prioritize.